src/main/java/io/knotx/junit5/KnotxTestUtils.java

```diff
@@ -15,16 +15,16 @@
  */
 package io.knotx.junit5;
 
-import com.google.common.io.Resources;
+import io.knotx.junit5.util.FileReader;
 import io.reactivex.Observable;
 import io.vertx.core.http.HttpMethod;
 import io.vertx.reactivex.core.http.HttpClient;
 import io.vertx.reactivex.core.http.HttpClientRequest;
 import io.vertx.reactivex.core.http.HttpClientResponse;
 import java.io.IOException;
-import java.nio.charset.StandardCharsets;
 import java.util.function.Consumer;
 
+@Deprecated
```

Comment:
Let's either remove it or add note to use `io.knotx.junit5.util.FileReader` instead.

Reply:
Note added to method